### DISCUSSION TITLE
What is the purpose of `scripting_enabled` and `iframe_srcdoc` in `swc_html_parser::parser::ParserConfig`?

### DISCUSSION THREAD

**Discussion by phoenix-ru** [swc-project/swc#7982] [Q&A] [answered]
I am working on a native Vue compiler (unofficial), and so far experience with `swc_ecma_parser` and `swc_ecma_codegen` in Rust has been amazing. I want to use `swc_html_parser` instead of a naive `nom` parser or `html5ever`.

Could you please advise, what is the point of `ParserConfig` in relation to SWC and parsing HTML? 
https://rustdoc.swc.rs/swc_html_parser/parser/struct.ParserConfig.html

```rs
pub struct ParserConfig {
    pub scripting_enabled: bool,
    pub iframe_srcdoc: bool,
}
```

There unfortunately is no documentation for these two fields, and I am not sure of their impact. Does this have anything to do with how `<script>` tags are parsed?

**Comment by kdy1** [comment]
I think it's for parity with the specification, but let's check again.
 cc @alexander-akait

**Reply by alexander-akait** [reply, marked as answer]
@phoenix-ru Sorry for delay, it is for parsing the `srcdoc` attribute of the `iframe` tag https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement/srcdoc, yeah it is a part of HTML spec
